kolibri/plugins/learn/assets/src/views/TopicsPage/TopicRow.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
I can't comment on it because it's empty ^^ but why the empty file?

Reply:
:man_shrugging:

Reply:
You commented later on in this file so I'm wondering if you hit a bug in Github / VS Code or something - or maybe I'm missing something

Reply:
Ah - weird - GH says your comment was on TopicsPage/TopicRow.vue - thanks for clarifying, will remove the empty.###